packages/dd-trace/test/appsec/rasp/ssrf.express.plugin.spec.js

```diff
@@ -162,6 +162,49 @@ describe('RASP - ssrf', () => {
           })
         })
       })
+
+      describe('Test using request', () => {
+        withVersions('express', 'request', requestVersion => {
+          let requestToTest
+
+          beforeEach(() => {
+            requestToTest = require(`../../../../../versions/request@${requestVersion}`).get()
+          })
+
+          it('Should not detect threat', async () => {
+            app = (req, res) => {
+              requestToTest.get(`https://${req.query.host}`).on('response', () => {
```

Comment:
why is this not a threat btw ? the domain is directly coming from the user inputs no ?

Reply:
the rasp doesn't detect the threat because we were sending an address with an incomplete uri. 
It should be `https://localhost/ifconfig.pro` but is `localhost` so it doesn't trigger the rasp rule

Reply:
oh ok ok